Interop.md

```diff
@@ -23,3 +23,8 @@ Creating snapshots of a VM which has containers running is not supported.
 - If a container is running and a snapshot is taken then if the container exits, the volume is not detached. VC shows an error that the volume is part of a snapshot and hence can't be removed from the VM config. For this reason, snapshotting a VM after starting one or more containers isn't supported as it leaves the volume attached to the VM. [#1649](https://github.com/vmware/docker-volume-vsphere/issues/1649)
 
 
+**Site Recovery Manager[SRM]**
+
+User created vmgroup is not supported with SRM.
+
+- After disaster recovery finishes, 'admin cli volume ls' command lists vmgroup for volumes of user created vmgroup as 'N/A' at secondary site and all VMs that were part of user created vmgroup will become part of default vmgroup and will have access to volumes of default vmgroup. [#1786](https://github.com/vmware/docker-volume-vsphere/issues/1786)
```

Comment:
After disaster recovery finishes, `esxcli volume ls` shows *N/A* (volume<->vmgroup relation) for the volumes part of user created vmgroup at the secondary site and all VMs that were part of user crated vmgroup will fall into `_DEFAULT` vmgroup  and gain access to all volumes in short it looses granted privileges and set resource consumption and resulting into full access or access right set for `_DEFAULT` vmgroup.

How about this?